src/api/app/models/bs_request.rb

```diff
@@ -949,24 +949,14 @@ def self.delayed_auto_accept
 
   # Check if 'user' is maintainer in _all_ request targets:
   def is_target_maintainer?(user = User.current)
-    has_target, is_target_maintainer = false, true
+    matched = false
     bs_request_actions.each do |a|
       next unless a.target_project
-      if a.target_package
-        tpkg = Package.find_by_project_and_name(a.target_project, a.target_package)
-        if tpkg
-          has_target = true
-          is_target_maintainer &= user.can_modify_package?(tpkg)
-          next
-        end
-      end
-      tprj = Project.find_by_name(a.target_project)
-      if tprj
-        has_target = true
-        is_target_maintainer &= user.can_modify_project?(tprj)
-      end
+      # avoid to become target maintainer if no target is defined.
```

Comment:
become is maybe not the right term ... "avoid to state to be a target maintainer in case the target does not exist actually".